addon/index.js

```diff
@@ -24,26 +28,19 @@ const mod = new PageMod({
   onAttach: setupApp
 });
 
-if (!store.experiments) {
-  store.experiments = {};
-}
-
-if (!store.availableExperiments) {
-  store.availableExperiments = [];
+if (!store.clientUUID) {
+  store.clientUUID = generateUUID();
 }
 
 require('sdk/system/unload').when(function(reason) {
   if (reason === 'uninstall') {
     app.send('addon-self:uninstalled');
-    if (store.experiments) {
-      for (let ex in store.experiments) { // eslint-disable-line prefer-const
-        if (store.experiments.hasOwnProperty(ex)) {
-          uninstall({addon_id: ex});
-        }
+    if (store.installedAddons) {
+      for (let id of store.installedAddons) { // eslint-disable-line prefer-const
```

Comment:
I want to say I used `let` because `const id` didn't work, but that might be a Chrome / V8 / node bug.